File: Syncfusion_Sample/Windows/XlsIO.Windows/Samples/2.0/Data Binding/Template Marker/VB/Customer.vb
Imports System.Drawing
Namespace EssentialXlsIOSamples
    Class Customer

#Region "Members"
        Private m_salesPerson As String
        Private m_salesJanJune As String
        Private m_salesJulyDec As String
        Private m_change As Integer
#End Region

#Region "Prperties"
        Public Property SalesPerson() As String
            Get
                Return m_salesPerson
            End Get
            Set(ByVal value As String)
                m_salesPerson = value
            End Set
        End Property

        Public Property SalesJanJune() As String
            Get
                Return m_salesJanJune
            End Get
            Set(ByVal value As String)
                m_salesJanJune = value
            End Set
        End Property
        Public Property SalesJulyDec() As String
            Get
                Return m_salesJulyDec
            End Get
            Set(ByVal value As String)
                m_salesJulyDec = value
            End Set
        End Property

        Public Property Change() As Integer
            Get
                Return m_change
            End Get
            Set(ByVal value As Integer)
                m_change = value
            End Set
        End Property

#End Region

#Region "Intialization"
        Public Sub New()
        End Sub
        Public Sub New(ByVal name As String, ByVal juneToJuly As String, ByVal julyToDec As String, ByVal change As Integer)
            Me.m_salesPerson = name
            Me.m_salesJanJune = juneToJuly
            Me.m_salesJulyDec = julyToDec
            Me.m_change = change
        End Sub
#End Region

    End Class
End Namespace

File: Syncfusion_Sample/Windows/XlsIO.Windows/Samples/2.0/Data Binding/Template Marker/CS/Customer.cs
﻿
using System.Drawing;
namespace EssentialXlsIOSamples
{
    class Customer
    {
        #region Members
        private string m_salesPerson;
        private string m_salesJanJune;
        private string m_salesJulyDec;
        private int m_change;
        #endregion

        #region Prperties
        public string SalesPerson
        {
            get
            {
                return m_salesPerson;
            }
            set
            {
                m_salesPerson = value;
            }
        }

        public string SalesJanJune
        {
            get
            {
                return m_salesJanJune;
            }
            set
            {
                m_salesJanJune = value;
            }
        }
        public string SalesJulyDec
        {
            get
            {
                return m_salesJulyDec;
            }
            set
            {
                m_salesJulyDec = value;
            }

        }
        public int Change
        {
            get
            {
                return m_change;
            }
            set
            {
                m_change = value;
            }

        }
        #endregion

        #region Intialization
        public Customer()
        {
        }
        public Customer(string name,string juneToJuly,string julyToDec,int change)
        {
            this.m_salesPerson = name;
            this.m_salesJanJune = juneToJuly;
            this.m_salesJulyDec = julyToDec;
            this.m_change = change;
        }
        #endregion
    }
}
